server/lyft/decorators/destroy_plan_step_runner_wrapper.go

```diff
@@ -0,0 +1,21 @@
+package decorators
+
+import (
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+)
+
+const PlanMode = "plan_mode"
```

Comment:
Let's name this a bit more descriptive? Maybe `deprecated` or `marked_for_deletion`?

Reply:
changed it to `deprecated`